### ISSUE TITLE
comparison with libmpack

### ISSUE THREAD

**Issue by justinmk** [ludocode/mpack#101]
There's another project (since 2016) with a similar name: https://github.com/libmpack/libmpack

Would be useful to see it in your comparison table:

https://github.com/ludocode/mpack#comparison-with-other-parsers

I also wonder if the names are too similar? It looks like https://github.com/ludocode/mpack generates its library file as `libmpack`, which may conflict with https://github.com/libmpack/libmpack .

**Comment by ludocode** [comment]
Yes, unfortunately the projects have similar names. If you look at the first commit of each project you'll find that this project came first and "libmpack" project came second, but only by a few months. This project is also more popular so I'm not about to rename it. To differentiate them I refer to this project as "MPack" and that other one as "libmpack".

MPack does not generate its library file as libmpack; it does not generate a library file at all. It is a source-only library, which means it does not get compiled separately. It is designed to be compiled directly into a project, not as a standalone library. Some people have forked MPack to add a buildsystem that compiles it into a library but I have resisted merging any of these into the official MPack.

You're right that I should add libmpack to the comparison table and benchmarks. I will leave this open until it's done.
